migrations/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM openjdk:8-jre-alpine
+
+RUN apk --no-cache add openssl
+
+WORKDIR /flyway
+
+ENV FLYWAY_VERSION 4.2.0
+
+RUN wget https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_VERSION}.tar.gz \
+  && tar -xzf flyway-commandline-${FLYWAY_VERSION}.tar.gz \
+  && mv flyway-${FLYWAY_VERSION}/* . \
+  && rm flyway-commandline-${FLYWAY_VERSION}.tar.gz \
+  && sed -i 's/bash/sh/' /flyway/flyway \
+  && ln -s /flyway/flyway /usr/local/bin/flyway
+
+COPY migrations/sql /flyway/sql/
+COPY migrations/flyway.conf /flyway/conf/flyway.conf
+
```

Comment:
baselineOnMigrate required because the orderly db is presumed to pre-exist and be non-empty when we first come along to do our migration to add the orderlyweb tables